quickstarts/Tuning.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # train_dataset=pathlib.Path('/content/data.csv')
</pre><p>nit: drop the <code>/content</code> part of the path - that is only true on Colab. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Done